isofit/radiative_transfer/radiative_transfer.py

```diff
@@ -192,21 +192,9 @@ def calc_rdn(self, x_RT, rfl_dir, rfl_dif, Ls, geom):
         s_alb = r["sphalb"]
 
         # flux coupling terms
```

Comment:
"Radiances along all optical paths" might be the more precise definition here? Typically "flux" refers to photons per area per time, e.g. an irradiance.

Reply:
Another good point! Updated with commit [665f264](https://github.com/isofit/isofit/pull/524/commits/665f264cc3be3b7ed2a13d80d719e1e2a95f76a1).